### PR TITLE
New theme colors

### PR DESCRIPTION
**Pull request by ydirson**

### CODE REVIEW

**Comment by stormi** [comment]
As we modify the patch, we can rename it to indicate that we changed it when xsconsole was in version 10.1.13.

Can you also bump the release and add a changelog entry so that we can build directly from your commit?

**Comment by olivierlambert** [comment]
The rest is the same on the host we used to make the test :)

File: SPECS/xsconsole.spec [hunk 1]
```diff
@@ -54,6 +54,10 @@ Console tool for configuring a XCP-ng installation.
 %{_unitdir}/xsconsole.service
 
 %changelog
+* Tue Aug 30 2022 Yann Dirson yann.dirson@vates.fr> - 10.1.13-1.2
```
Comment (stormi) [inline SPECS/xsconsole.spec:57]: This is not exactly the date for today, and `<` is missing at the beginning of the e-mail address.

Comment (ydirson) [reply SPECS/xsconsole.spec:57]: I miss the emacs mode for .spec edition ;)

**Review by stormi — APPROVED** [review]

**Pull request merged**